core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -1136,4 +1138,18 @@ public FeaturesConfig setIncrementalHashArrayLoadFactorEnabled(boolean increment
         this.incrementalHashArrayLoadFactorEnabled = incrementalHashArrayLoadFactorEnabled;
         return this;
     }
+
+    @NotNull
```

Comment:
nit. This annotation does not make sense. Primitives cannot be null.

Reply:
ok